packages/kamino-sdk/src/services/OrcaService.ts

```diff
@@ -68,6 +74,19 @@ export class OrcaService {
     if (!pool || !whirlpool) {
       throw Error(`Could not get orca pool data for ${strategy.pool}`);
     }
+    const priceRange = this.getStrategyPoolPriceRange(position, strategy, pool);
+    if (priceRange.strategyOutOfRange) {
+      return {
+        ...priceRange,
+        rewardsApy: [],
+        rewardsApr: [],
+        feeApy: ZERO,
+        feeApr: ZERO,
+        totalApy: ZERO,
```

Comment:
is this meant to represent pool + kamino APY or just pool APY?

Reply:
This is just pool APY